src/scss/vendor/_tom-select.scss

```diff
@@ -3,6 +3,11 @@ $input-border-width: 1px;
 
 @import "~tom-select/src/scss/tom-select.bootstrap5.scss";
 
+:root {
+  --ts-pr-clear-button: 0rem;
+  --ts-pr-caret: 0rem;
+}
+
```

Comment:
This will not be necessary as soon as Tom Select merges https://github.com/orchidjs/tom-select/pull/530.